pkg/standardgadgets/trace/standardtracerbase.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```

Comment:
Why this file is not directly in pkg/standardgadgets if it is also used by profile gadgets?

Reply:
It's only used by trace gadgets:

```
 $ git grep NewStandardTracer
pkg/standardgadgets/trace/bind/tracer.go:       baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
pkg/standardgadgets/trace/capabilities/tracer.go:       baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
pkg/standardgadgets/trace/exec/tracer.go:       baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
pkg/standardgadgets/trace/mount/tracer.go:      baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
pkg/standardgadgets/trace/open/tracer.go:       baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
pkg/standardgadgets/trace/standardtracerbase.go:func NewStandardTracer(lineCallback func(string), mntnsmap *ebpf.Map, name string,
pkg/standardgadgets/trace/tcp/tracer.go:        baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
pkg/standardgadgets/trace/tcpconnect/tracer.go: baseTracer, err := trace.NewStandardTracer(lineCallback, config.MountnsMap,
```

Reply:
I don't know what I see, maybe again my environment wasn't clear 😅